src/components/bottom-bar.tsx

```diff
@@ -22,46 +22,48 @@ export class BottomBarComponent extends BaseComponent<IProps, IState> {
     const primaryMag = simulation.getMagnetAtIndex(0);
     const secondaryMag = simulation.getMagnetAtIndex(1);
     const showMagForces: boolean = primaryMag != null && secondaryMag != null;
-    const forcesLabel: string = simulation.showMagneticForces ? "ON" : "OFF";
-    const forcesOn: boolean = forcesLabel === "ON" ? true : false;
     if (!primaryMag) {
-      return (null);
+      return (
+        <div className="bottom-bar">
+          {this.renderLogo()}
+        </div>
+      );
     } else {
       return (
         <div className="bottom-bar">
-          <div className="row">
-            <div className="panel">
-              <PolarityPanelComponent index={0}/>
-              <StrengthPanelComponent index={0}/>
+          <PolarityPanelComponent index={0}/>
```

Comment:
Not worth changing, but I'm wondering if it would have been cleaner for us to just pass the magnet down to the component here since we already have it...

Reply:
Hmmm.... not a bad thought.  Might have been the better way.